src/test/fuzz/util.h

```diff
@@ -296,7 +296,6 @@ template <bool ReturnUniquePtr = false>
 auto ConsumeNode(FuzzedDataProvider& fuzzed_data_provider, const std::optional<NodeId>& node_id_in = std::nullopt) noexcept
 {
     const NodeId node_id = node_id_in.value_or(fuzzed_data_provider.ConsumeIntegralInRange<NodeId>(0, std::numeric_limits<NodeId>::max()));
-    const ServiceFlags local_services = ConsumeWeakEnum(fuzzed_data_provider, ALL_SERVICE_FLAGS);
```

Comment:
why is this removed and replaced with a hardcoded value?

Reply:
Which hardcoded value? The `CNode` object no longer contains `nServices`, so there's nothing to set here.

Reply:
The symbol is moved from `CNode` to `Peer`. I think a refactoring pull request should not change the value of the symbol.

If the local service flags were previously determined by the fuzz engine, they should still be after this change, regardless of the namespace of the symbol.

Reply:
To be clear, are you suggesting that the `InitializeNode()` calls in fuzz/processmessage.cpp and fuzz/processmessages.cpp should be updated to take `ConsumeWeakEnum(fuzzed_data_provider, ALL_SERVICE_FLAGS)`?

Reply:
Yes.

I am also thinking if `InitializeNode` can be moved into `FillNode`, so that only one place needs to be updated.

Reply:
Moved all `InitializeNode` calls (there were only 2) into `FillNode` with service flags consumed from the fuzzer.